src/Bicep.Core.Samples/Files/ModulesWithScopes_LF/main.json

```diff
@@ -1,279 +1,493 @@
+                          "resources": [],
+                          "outputs": {
+                            "myOutput": {
+                              "type": "string",
+                              "value": "hello!"
+                            }
+                          }
+                        }
+                      }
+                    }
+                  ],
+                  "outputs": {
+                    "myOutput": {
+                      "type": "string",
+                      "value": "[reference(tenantResourceId('Microsoft.Resources/deployments', 'myTenantMod'), '2019-10-01').outputs.myOutput.value]"
+                    },
+                    "myOutputResourceGroup": {
+                      "type": "string",
+                      "value": "[reference(extensionResourceId(format('/subscriptions/{0}/resourceGroups/{1}', 'db90cfef-a146-4f67-b32f-b263518bd216', 'myOtherRg'), 'Microsoft.Resources/deployments', 'myOtherResourceGroup'), '2019-10-01').outputs.myOutput.value]"
+                    }
+                  }
+                }
+              }
+            },
+            {
+              "type": "Microsoft.Resources/deployments",
+              "apiVersion": "2019-10-01",
+              "name": "myTenantMod",
+              "scope": "/",
+              "properties": {
+                "expressionEvaluationOptions": {
+                  "scope": "inner"
+                },
+                "mode": "Incremental",
+                "template": {
+                  "$schema": "https://schema.management.azure.com/schemas/2019-08-01/tenantDeploymentTemplate.json#",
+                  "contentVersion": "1.0.0.0",
+                  "functions": [],
+                  "resources": [],
+                  "outputs": {
+                    "myOutput": {
+                      "type": "string",
+                      "value": "hello!"
+                    }
+                  }
+                }
+              }
+            }
+          ],
+          "outputs": {
+            "myOutput": {
+              "type": "string",
+              "value": "[reference(tenantResourceId('Microsoft.Resources/deployments', 'myTenantMod'), '2019-10-01').outputs.myOutput.value]"
+            },
+            "myOutputRgMod": {
+              "type": "string",
+              "value": "[reference(extensionResourceId(format('/subscriptions/{0}/resourceGroups/{1}', subscription().subscriptionId, 'myRg'), 'Microsoft.Resources/deployments', 'myResourceGroupMod'), '2019-10-01').outputs.myOutput.value]"
+            },
+            "myOutputRgMod2": {
+              "type": "string",
+              "value": "[reference(extensionResourceId(format('/subscriptions/{0}/resourceGroups/{1}', subscription().subscriptionId, 'myRg'), 'Microsoft.Resources/deployments', 'myResourceGroupMod2'), '2019-10-01').outputs.myOutput.value]"
+            }
+          }
+        }
+      }
+    },
+    {
+      "condition": "[equals(length('foo'), 3)]",

```

Comment:
If we do support runtime functions, it'd be good to add a case.

Reply:
Confirmed that runtime functions are not supported.